pkg/kubectl/describe.go

```diff
@@ -107,17 +105,8 @@ func DescribableResources() []string {
 
 // Describer returns the default describe functions for each of the standard
 // Kubernetes types.
-func DescriberFor(group string, kind string, c *client.Client) (Describer, bool) {
-	var f Describer
-	var ok bool
-
-	switch group {
-	case "":
-		f, ok = describerMap(c)[kind]
-	case "extensions":
-		f, ok = expDescriberMap(c)[kind]
-	}
-
+func DescriberFor(kind unversioned.GroupKind, c *client.Client) (Describer, bool) {
+	f, ok := describerMap(c)[kind]
```

Comment:
> return describerMap(c)[kind]

That won't compile.  the `value, ok` return is some sort of special cased thing.  When done with a return, its just the `value`